integrate/data-streams/overview/index.md

```diff
@@ -54,45 +71,45 @@ The `template` column type currently only supports the `jsonnet` templating lang
 
-## Update Receivers and Exports
+## Update Sources and Sinks
 
-A receiver may be updated using the [`PATCH /v1/receivers/{receiverId}`](https://buffer.keboola.com/v1/documentation/#/configuration/UpdateReceiver) endpoint. Exports maybe updated using the [`PATCH /v1/receivers/{receiverId}/exports/{exportId}`](https://buffer.keboola.com/v1/documentation/#/configuration/UpdateExport) endpoint.
+A source may be updated using the [`PATCH /v1/branches/{branchId}/sources/{sourceId}`](https://stream.keboola.com/v1/documentation/#/configuration/UpdateSource) endpoint. Sinks maybe updated using the [`PATCH /v1/branches/{branchId}/sources/{sourceId}/sinks/{sinkId}`](https://stream.keboola.com/v1/documentation/#/configuration/UpdateSink) endpoint.
 
-The `UpdateReceiver` endpoint may only update the receiver's name. Exports may only be updated separately.
+The `UpdateSource` endpoint may only update the source's name. Sinks may only be updated separately.
 
-If an export's `mapping.tableId` is updated, it is handled the same way as in the create operation. If the table exists, `mapping.columns` must match the existing table's schema. If the table does not exist, it is created.
+If a sink's `mapping.tableId` is updated, it is handled the same way as in the create operation. If the table exists, `mapping.columns` must match the existing table's schema. If the table does not exist, it is created.
 
 ## Tokens
 
-A token is generated for each receiver export. These tokens have the minimum possible scope, which is a `write` permission for the bucket in which the destination table is stored. You can see these tokens at `https://connection.keboola.com/admin/projects/<project-id>/tokens-settings`. Their description is in the format `[_internal] Buffer Export <export-id> for Receiver <receiver-id>`.
+A token is generated for each source sink. These tokens have the minimum possible scope, which is a `write` permission for the bucket in which the destination table is stored. You can see these tokens at `https://connection.keboola.com/admin/projects/<project-id>/tokens-settings`. Their description is in the format `[_internal] Stream Sink <sink-id> for Source <source-id>`.
 
-These tokens should not be deleted or refreshed manually. To refresh tokens, use the [`POST /v1/receivers/{receiverId}/tokens/refresh`](https://buffer.keboola.com/v1/documentation/#/configuration/RefreshReceiverTokens) endpoint.
+These tokens should not be deleted or refreshed manually. To refresh tokens, use the [`POST /v1/branches/{branchId}/sources/{sourceId}/tokens/refresh`](https://stream.keboola.com/v1/documentation/#/configuration/RefreshSourceTokens) endpoint.
```

Comment:
I think we do not have anymore refresh tokens. Also we have new settings per `source` or `sink`. Clear statistics should be mentioned? Also aggregation endpoint is only internal
New docs: 
https://stream.eu-west-1.aws.keboola.dev/v1/documentation/#/

Old docs:
https://buffer.keboola.com/v1/documentation/#/

Also there is test endpoint. Should be there?